pkg/interactor/delete_data_test.go

```diff
@@ -0,0 +1,112 @@
+package interactor
+
+import (
+	"errors"
+	"testing"
+
+	thingentities "github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+type DeleteDataTestCase struct {
+	name               string
+	authorization      string
+	deviceID           string
+	expectedDataError  error
+	expectedThing      []*thingentities.Thing
+	expectedThingError error
+	fakeLogger         *mocks.FakeLogger
+	fakeDataStore      *mocks.FakeDataStore
+	fakeService        *mocks.FakeService
+}
+
+var ddCases = []DeleteDataTestCase{
+	{
+		"authorization key not provided",
```

Comment:
Please, test if `deviceID` is provided too.

Reply:
Done!